src/components/Navbar/Navbar.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styles from './Navbar.module.css';
+import Navbar from 'react-bootstrap/Navbar';
 import web from 'assets/talawa-logo-lite-200x200.png';
-function Navbar(): JSX.Element {
+function NavBar(): JSX.Element {
   return (
-    <nav className={styles.nav}>
-      <div className={styles.main}>
-        <a className={styles.logo} href="/">
-          <img src={web} />
-          <strong>Talawa</strong>
-          <strong>Admin</strong>
-        </a>
-      </div>
-    </nav>
+    <div>
+      <Navbar bg="">
+        <Navbar.Brand>
+          <a className={styles.logo} href="/">
+            <img src={web} />
+            <strong>Talawa</strong>
+            <strong>Admin</strong>
+          </a>
+        </Navbar.Brand>
+      </Navbar>
+    </div>
```

Comment:
For all of the components so far, React fragments, were used for wrapping the components, so please make use of them to ensure that there is consistency
```suggestion
    <>
      <Navbar bg="">
        <Navbar.Brand>
          <a className={styles.logo} href="/">
            <img src={web} />
            <strong>Talawa</strong>
            <strong>Admin</strong>
          </a>
        </Navbar.Brand>
      </Navbar>
    </>
```

Reply:
@DangaRanga On it !